lib/publishes_to_publishing_api.rb

```diff
@@ -12,10 +13,14 @@ def can_publish_to_publishing_api?
   end
 
   def publish_to_publishing_api
-    Whitehall::PublishingApi.publish_async(self)
+    run_callbacks :published do
```

Comment:
`StatisticsAnnouncement` updates the search index in a callback on publish now which is the intended behaviour